src/features/exporter/js/exporter.js

```diff
@@ -1535,7 +1535,7 @@
             // The standard column width in Microsoft Excel 2000 is 8.43 characters based on fixed-width Courier font
             // Width of 10 in excel is 75 pixels
             var colWidths = [];
-            var startDataIndex = grid.treeBase ? grid.treeBase.numberLevels : 0;
+            var startDataIndex = grid.treeBase ? grid.treeBase.numberLevels : (grid.enableRowSelection !== false ? 1 : 0);
```

Comment:
Why are you checking for different than false rather than simply check for a truthy value?

```
grid.enableRowSelection? 1 : 0
```

Reply:
because it may be null. I was following the pattern already in the code.

Reply:
line 361 selection.js

          gridOptions.enableRowSelection = gridOptions.enableRowSelection !== false;

Reply:
Well, that line in 361, ensures that enableRowSelection is a boolean from that point on. I don't believe there is ever a way to get into this function without running through 361 first. But if you feel strongly about keeping it this way, I won't hold your pull request because of it.

Reply:
Never mind, I decided to separate your doc changes from your bug fix by doing a cherry pick and creating a new PR for the fix alone. I will add unit tests and merge it on its own, Then we can address the doc changes together.